src/Command/InfectionCommand.php

```diff
@@ -255,7 +257,7 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
         }
 
         $this->consoleOutput = $this->getApplication()->getConsoleOutput();
-        $this->skipCoverage = \strlen(trim($input->getOption('coverage'))) > 0;
+        $this->skipCoverage = '' !== trim((string) $input->getOption('coverage'));
```

Comment:
I changed that since I find it easier to read, but maybe you don't agree on it or it's not the convention in the project, you tell me

Reply:
👍 for that change